Tasks/AzureCLI/task.json

```diff
@@ -0,0 +1,104 @@
+{
+  "id": "46E4BE58-730B-4389-8A2F-EA10B3E5E815",
+  "name": "AzureCLI [Preview]",
+  "friendlyName": "Azure CLI preview",
+  "description": "Run a Shell script with Azure CLI commands against an azure subscription",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "Replace with markdown to show in help",
```

Comment:
I have asked the PM to write a the readme file for this, they might go with the readme file or by writing a blog. As tasks use one of the two ways.